src/RequestInterface.php

```diff
@@ -55,10 +57,10 @@ public function getRequestTarget();
      *
      * @link http://tools.ietf.org/html/rfc7230#section-5.3 (for the various
      *     request-target forms allowed in request messages)
-     * @param mixed $requestTarget
+     * @param string $requestTarget
```

Comment:
Was this an error before?

Reply:
Yes - if you look at the linked RFC 7230 section, you'll see it can only be a string.

Reply:
refs #78